src/js/bellows.js

```diff
@@ -52,6 +52,12 @@
             this._bindEvents();
         },
 
+        destroy: function() {
+            this.$bellows
+                .removeData(this.name)
+                .off(this.options.event);
```

Comment:
Should call off with the function that we want to unbind. Otherwise all handlers that are bound to $bellows of this type will get nuked.